### DISCUSSION TITLE
API call /v1/analysis dont show suppressed findings

### DISCUSSION THREAD

**Discussion by JoergBruenner** [DependencyTrack/dependency-track#2629] [Q&A] [answered]
Hi all,

I'd like to see all my audit findings. The call

curl -X GET "http://localhost:8081/api/v1/analysis?project=<UUID>&component=9<UUID>&vulnerability=<UUID>" -H  "accept: application/json" -H  "X-Api-Key: <API-Key>"

works in general. I shows only non suppressed findings. What do I have to change to see all findings?

Thanks a lot for your help
Jörg

**Comment by Souhila99** [comment, marked as answer]
Add to the query the parameter suppressed=true

**Reply by JoergBruenner** [reply]
Thank you, it works :-) Where can I learn things like this, if swagger.json won't tell it?
